app/src/main/java/org/bspb/smartbirds/pro/ui/map/OsmMapProvider.java

```diff
@@ -184,6 +186,11 @@ public void setShowZoneBackground(boolean showBackground) {
         drawZones();
     }
 
+    @Override
+    public void setShowLocalProjects(boolean showKml) {
+
+    }
```

Comment:
not implemented?

Reply:
Will implement after finish with OSM dependency upgrade